### ISSUE TITLE
Row broken has cell with text that leaks

### ISSUE THREAD

**Issue by pirasalbe** [simonbengtsson/jsPDF-AutoTable#491]
Hi,
I've a table with a row that breaks in another page, as you can see in the image, and part of the text goes out the margin of its cell.

![image](https://user-images.githubusercontent.com/4697821/56278313-4a8d8f00-6106-11e9-9884-24ac397b1165.png)

I have this behaviour since v3.0.9.

**Code and variables:**

```
doc.autoTable({
      columns: columns,
      body: rows,
      theme: 'plain',
      styles: {
      "cellPadding": 2,
      "fillColor": false,
      "lineColor": 200,
      "lineWidth": 0.5,
      "overflow": "linebreak",
      "cellWidth": "auto"
    },
      headStyles: {
      "font": "helvetica",
      "fontSize": 7,
      "fillColor": 200,
      "halign": "left",
      "valign": "middle",
      "textColor": 20,
      "fontStyle": "bold"
    },
      columnStyles: columnsStyles,
      startY: 683.1999999999998,
      margin: {
        top: 40,
        right: 20,
        left: 45,
        bottom: 20
      },
      tableWidth: 555.28,
      showHead: 'firstPage'
    });
```

> The char ↵ means \n

```
const rows = [{
beneficiario: "BENEFICIARIO"
datiBancari: "IBAN:↵ITAAA4523462CFERVBE↵ABI: F4F4↵CAB: 333↵BIC: GDT"
datiGenerali: "Modalità: bonifico (MP05)↵Cod. pag.: R4R4↵Data rif.: 30/03/2018↵Gg term.: 30↵Data scad.: 29/04/2018↵Uff. post.: DED3↵Ist. fin.: TEST"
importo: "14.016,53"
pagamentoAnticipato: "Data lim.: 29/04/2018↵Sconto: 5,00"
pagamentoRitardato: "Data dec.: 29/04/2018↵Penalità: 100,00"
quietanzante: "MARIO ROSSI↵AAABBB45OJ533534↵Titolo: MR"
}],[{
beneficiario: "BENEFICIARIO 3"
datiBancari: "IBAN:↵ITAAA4523462CFERVBE↵ABI: F4F4↵CAB: 333↵BIC: GDT"
datiGenerali: "Modalità: bonifico (MP05)↵Cod. pag.: R4R4↵Data rif.: 30/03/2018↵Gg term.: 30↵Data scad.: 29/04/2018↵Uff. post.: DED3↵Ist. fin.: TEST"
importo: "50,53"
pagamentoAnticipato: "Data lim.: 29/04/2018↵Sconto: 5,00"
pagamentoRitardato: "Data dec.: 29/04/2018↵Penalità: 100,00"
quietanzante: "MARIO ROSSI↵AAABBB45OJ533534↵Titolo: MR"
}]
```

```
const columns = [
{header: "Beneficiario", dataKey: "beneficiario"}, 
{header: "Dati pagamento", dataKey: "datiGenerali"},
{header: "Quietanzante", dataKey: "quietanzante"}, 
{header: "Dati bancari", dataKey: "datiBancari"}, 
{header: "Pag. ant.", dataKey: "pagamentoAnticipato"}, 
{header: "Pag. rit.", dataKey: "pagamentoRitardato"}, 
{header: "Importo pag.", dataKey: "importo"}
]
```

**Comment by pirasalbe** [comment]
I think that this issue is related to #469

**Comment by simonbengtsson** [comment]
Can't reproduce, can you create a minimal example highlighting the issue?

**Comment by simonbengtsson** [comment]
Probably fixed in v3.2.3

**Comment by pirasalbe** [comment]
It's now fixed, thank you
